plugins/fclrave/fclspace.h

```diff
@@ -265,12 +296,17 @@ class FCLSpace : public boost::enable_shared_from_this<FCLSpace>
             else {
                 const std::vector<KinBody::Link::GeometryPtr> & vgeometries = plink->GetGeometries();
                 FOREACH(itgeom, vgeometries) {
+                    const KinBody::GeometryPtr& pgeom = *itgeom;
```

Comment:
``itgeom`` is dereferenced twice unneessarily, let's avoid it by using range based for loop (``for (const KinBody::GeometryPtr& pgeom : vgeometries)``), it's faster than iterator based.

Reply:
In order to set `enclosingBV` for the first time in L318 (originally L282), itgeom is used.
Should we compare pgeom and vgeommetries[0], or should we rewrite L300 (originally L268) as
```
const KinBody::GeometryInfo& geominfo = pgeom->GetInfo();
```
?

Reply:
then let's keep ``itgeominfo``, and make a new variable to store const reference to ``*pgeominfo``. Currently, ``pgeominfo`` is deferenced three times by ``*`` and ``->`` operators which is time consuming.